projects/valtimo/components/src/lib/components/breadcrumb-navigation/breadcrumb.service.ts

```diff
@@ -75,6 +83,15 @@ export class BreadcrumbService {
 
   clearSecondBreadcrumb(): void {
     this._manualSecondBreadcrumb$.next(null);
+    this._manualThirdBreadcrumb$.next(null);
```

Comment:
Change to clearThirdBreadcrumb(). Perhaps change clearSecondBreadcrumb to clearManualBreadcrumbs